docs/man_pages/screenbuilder/add-signin.md

```diff
@@ -0,0 +1,21 @@
+create
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder add-signin`
```

Comment:
should you have one "g" at the end - `signing`? Also is the command with `-` really :sob: